lib/index.js

```diff
@@ -140,18 +258,42 @@ Attribution.prototype.onalias = function(alias) {
  * @param {Object} msg
  */
 
-Attribution.prototype.normalize = function(msg) {
+Attribution.prototype.normalize = function(message) {
+  var msg = message;
   this.debug('normalize %o', msg);
   var user = this.analytics.user();
   var global = exports.global;
   var query = global.location.search;
-  var ctx = msg.context = msg.context || msg.options || {};
+  var ctx = (msg.context = msg.context || msg.options || {});
   delete msg.options;
+  msg.projectId = this.options.project || window.Attribution.projectId;
```

Comment:
@mgz I don't think we need this `window.Attribution.projectId`, that's probably an old hack which we don't need anymore.